# Pose from Ground-truth first pose and world origin (where
# the point-cloud is): this
# should be the first pose when the VIO starts
T_V_G:
- [0.3800,   0.1647 ,    0.9102, 0.878612]
- [0.0307,   -0.9857,   0.1655, 2.142470]
- [0.9245,    -0.0350,   -0.3796, 0.947262]
- [   0.0,       0.0,      0.0,      1.0]
invert_T_V_G: true

# For V1_01_easy
# This is the first pose of the robot
# x, y, z
# 0.878612, 2.142470, 0.947262

# q_w, q_x, q_y, q_z
# 0.060514,-0.828459,-0.058956,-0.553641

# Quat to Rot matrix
# 0.3800    0.0307    0.9245
# 0.1647   -0.9857   -0.0350
# 0.9102    0.1655   -0.3796
